src/community/pom.xml

```diff
@@ -568,6 +570,25 @@
         <module>monitor-hibernate</module>
       </modules>
     </profile>
+    <profile>
+      <id>monitor-kafka</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <version>3.2.0</version>
```

Comment:
this version is already provided through pluginManagment of the parent pom:
https://github.com/geoserver/geoserver/blob/f5355bc54f1a09b2c429406002dddd09f2f5fac3/src/pom.xml#L1699-L1702

```suggestion
```